vitrina/resources/views.py

```diff
@@ -172,6 +175,7 @@ def form_valid(self, form):
             self.dataset.status = Dataset.HAS_DATA
             self.dataset.save()
 
+        set_comment(("Pridėtas naujas duomenų šaltinis \"%(title)s\".") % {"title": resource.lt_title()})
```

Comment:
Let's not use the old style of formatting, either:

```suggestion
        set_comment(f'Pridėtas naujas duomenų šaltinis "{resource.lt_title()}".')
```
or
```suggestion
        set_comment('Pridėtas naujas duomenų šaltinis "{title}".'.format(
            title=resource.lt_title()
        ))
```

Other places in this file as well.